### PR TITLE
Add + apply stylelint configuration to LESS

### PR DESCRIPTION
**Pull request by rsek**
Unfortunately, there isn't much in the way of Stylelint plugins for LESS, and the ones I found don't work well, so the config has a bunch of ugly overrides for the time being (the SCSS version is much cleaner in that respect).

This doesn't cover all rules, but it does cover most of the changes. The remainder will be handled in the SCSS migration pull.

### CODE REVIEW

File: src/module/vue/components/attr-box.vue [hunk 1]
```diff
@@ -18,26 +18,28 @@
 </template>
 
 <style lang="less" scoped>
-@import '../../../styles/mixins.less';
+@import (reference) '../../../styles/mixins.less';
+
 .stat {
-  &:before {
+  &::before {
```
Comment (ben) [inline src/module/vue/components/attr-box.vue:24]: Oh boy. Was this broken, or is this just a preferred style?

File: src/styles/styles.less [hunk 2]
```diff
@@ -2,38 +2,30 @@
...
-  --widget-stroke-width: 1;
-
-  --clock-fill-opacity-hover: 0.6;
-
+  --ironsworn-widget-stroke-width: 1;
+  --ironsworn-clock-fill-opacity-hover: 0.6;
   --ironsworn-border-radius-sm: 2px;
   --ironsworn-border-radius-md: 3px;
   --ironsworn-border-radius-lg: 5px;
-
```
Comment (ben) [inline src/styles/styles.less:12]: Oh, losing these is a bit sad, it made this wall of rules more readable. 😞

**Review by ben — APPROVED** [review]
I did a random sampling of changes here, but I like the approach, and most of the diff looks great. Do we want to do a stylelint as part of the [CI workflow](https://github.com/ben/foundry-ironsworn/blob/main/.github/workflows/ci.yml#L39)?

File: src/styles/styles.less (hunk 2 above)
Comment (rsek) [reply src/styles/styles.less:12]: hm, good point - i'll tweak the rules to make them friendlier to arbitrary line breaks

File: src/module/vue/components/attr-box.vue (hunk 1 above)
Comment (rsek) [reply src/module/vue/components/attr-box.vue:24]: presently both work in most browsers (and a i'm pretty sure it's fine on any browser foundry officially supports), but "::before" appears to be the current standard while ":before" is an older style: https://developer.mozilla.org/en-US/docs/Web/CSS/::before

File: src/styles/styles.less (hunk 2 above)
Comment (rsek) [reply src/styles/styles.less:12]: alternatively -- could add some comments, which would make the content more explicit and force an extra blank line 🤔

**Comment by ben** [comment]
✨

**Pull request merged**